src/main/java/com/redhat/quarkus/mandrel/collector/report/endpoints/ImageStatsResource.java

```diff
@@ -340,4 +344,20 @@ public Response toResponse(Exception exception) {
             return Response.status(code).entity(exceptionJson).build();
         }
     }
+
+    @GET
+    @RolesAllowed("token_read")
+    @Path("/schema")
+    @Produces(MediaType.APPLICATION_JSON)
+    public String schema() {
+        ObjectMapper mapper = new ObjectMapper();
+
+        JsonSchemaGenerator generator = new JsonSchemaGenerator(mapper);
+        try {
+            JsonSchema schema = generator.generateSchema(ImageStats.class);
+            return mapper.writerWithDefaultPrettyPrinter().writeValueAsString(schema);
```

Comment:
Pretty printing by default? So it's for humans to see, right?

Reply:
My use case was to copy it in order to feed it in another tool (Horreum) for validation of the input.

But I can see it being used mostly by humans, either to understand the schema or to copy paste it in some other place.

Reply:
I ended up making it return the unformatted schema. One can easily pipe it to `jq` to pretty print.